src/main/kotlin/org/opensearch/commons/alerting/model/IndexExecutionContext.kt

```diff
@@ -21,7 +21,8 @@ data class IndexExecutionContext(
     val updatedIndexNames: List<String>,
     val concreteIndexNames: List<String>,
     val conflictingFields: List<String>,
-    val docIds: List<String>? = emptyList()
+    val docIds: List<String>? = emptyList(),
+    val findingIds: List<String>? = emptyList()
```

Comment:
why are we adding this here?

Reply:
just to add `findingIds` to `IndexExecutionContext`.